generate_templates.py

```diff
@@ -207,24 +208,35 @@ def main():
     interfacemap = json.loads(DEVICEMAP_PATH.read_text())
 
     for vendor, skus in interfacemap["interfaceMap"].items():
+        models = set()
         for sku, devicemap in skus.items():
             actual_devicemap = resolve_alias(devicemap, interfacemap)
             if not actual_devicemap:
                 continue
-            template = generate_template(vendor, sku, actual_devicemap)
-            write_template(vendor, sku, template)
 
-def generate_template(vendor, sku, devicemap):
+            model = devicemap.get('displayModel') or sku
+
+            # make sure models are unique per vendor
+            if model in models:
+                print("Vendor {vendor} has duplicate model {model}")
+                sys.exit(1)
+
+            models.add(model)
+
+            template = generate_template(vendor, model, actual_devicemap)
+            write_template(vendor, model, template)
+
+def generate_template(vendor, model, devicemap):
     return {
-        "name": f"{vendor}-{sku}-Template",
-        "description": f"Adds a standalone {vendor} {sku} router: {devicemap.get('description', '')}",
+        "name": f"{vendor}-{model}-Template",
+        "description": f"Adds a standalone {vendor} {model} router: {devicemap.get('description', '')}",
```

Comment:
I know not a part of your change,  but can you conditionalize the `:` in the description? If there is not a description from the devicemap, it leaves a hanging colon.